editoast/src/views/paced_train.rs

```diff
@@ -137,6 +137,43 @@ async fn get_by_id(
     Ok(Json(paced_train))
 }
 
+/// Update a paced train
+#[utoipa::path(
+    put, path = "",
+    tag = "timetable,paced_train",
+    params(PacedTrainIdParam),
+    request_body = inline(PacedTrainBase),
+    responses(
+        (status = 200, body = PacedTrainResult, description = "Paced train have been updated")
+    )
+)]
+async fn update_paced_train(
+    State(db_pool): State<DbConnectionPoolV2>,
+    Extension(auth): AuthenticationExt,
+    Path(PacedTrainIdParam { id: paced_train_id }): Path<PacedTrainIdParam>,
+    Json(paced_train_base): Json<PacedTrainBase>,
+) -> Result<impl IntoResponse> {
```

Comment:
```suggestion
) -> Result<PacedTrainResult> {
```

Reply:
Why is this resolved since it's not been fixed?